src/main/java/seedu/address/storage/JsonAdaptedRecipe.java

```diff
@@ -26,6 +27,7 @@
     private final String name;
     private final String ingredients;
     private final String instructions;
+    private String calorie; //if i make it final the code breaks!
```

Comment:
if I make it final I get a null pointer man!

Reply:
once you implement my fix, you should be able to set it to final